pd/src/info.rs

```diff
@@ -56,9 +58,48 @@ impl Info {
         &self,
         query: abci::request::Query,
     ) -> Result<abci::response::Query, anyhow::Error> {
-        tracing::warn!(?query, "unhandled query");
+        tracing::info!(?query);
+
+        match query.path.as_str() {
+            "/jmt/key" => {
```

Comment:
note: should we call this something else? for reference, the cosmos SDK implements ABCI query interfaces separately for each module: the kv query for ibc is `store/ibc/key` for instance. Since we don't have modules, I think just having a single kv query interface is probably good, but maybe we don't want to call it `jmt` as to avoid binding the name to an implementation detail

Reply:
How about `/state/key/`?